egui-winit/src/epi.rs

```diff
@@ -289,7 +301,12 @@ impl EpiIntegration {
         self.egui_winit
             .handle_output(window, &self.egui_ctx, egui_output);
 
-        let app_output = self.frame.take_app_output();
+        let mut app_output = self.frame.take_app_output();
+        if app_output.drag_window {
+            app_output.drag_window &= self.can_drag_window;
+            self.can_drag_window = false;
```

Comment:
shouldn't this be reset each frame? Otherwise a single click will set `can_drag_window=true` and keep it set to `true` until `drag_window` is set

Reply:
Yes, you're right. I've tested with a reset on every frame, and it works fine.